README.md

```diff
@@ -362,12 +367,109 @@ The `$SERVICE_BINDING_ROOT` environment variable **MUST NOT** be reset if it is
 
 If a `.spec.type` is set, the `type` entry in the binding `Secret` **MUST** be set to its value overriding any existing value.  If a `.spec.provider` is set, the `provider` entry in the binding `Secret` **MUST** be set to its value overriding any existing value.
 
+### Binding Status
```

Comment:
I set this anchor here so I can reference it from the new extension, instead of duplicating text.

Reply:
How about renaming it to something like `Ready Condition Status`, since this section is focused specifically on the ready condition's status and not the whole binding status.

Reply:
good idea.  updated.